src/main/java/com/example/ajaxonboardingproject/controller/NatsController.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.ajaxonboardingproject.controller
+
+import com.google.protobuf.Parser
+import io.nats.client.Connection
+
+interface NatsController<T> {
```

Comment:
bound `T` to `GeneratedMessageV3`:
```kotlin
interface NatsController<ReqT: GeneratedMessageV3, RespT: GeneratedMessageV3> {

    fun handle(request: ReqT): RespT
}
```
I've replaced `T` with `ReqT` in my example

Reply:
Write me in private, I'll show you how to make your code better